src/registrar/utility/errors.py

```diff
@@ -23,6 +23,15 @@ class InvalidDomainError(ValueError):
     pass
 
 
+class OutsideOrgMemberError(ValueError):
```

Comment:
We can delete this error and all references to it once we allow users to be members of multiple orgs